Jellyfin.Plugin.Tvdb/TvdbUtils.cs

```diff
@@ -24,33 +25,65 @@ public static class TvdbUtils
         /// <param name="keyType">Key type.</param>
         /// <returns>Image type.</returns>
         /// <exception cref="ArgumentException">Unknown key type.</exception>
-        public static ImageType GetImageTypeFromKeyType(string keyType)
+        public static ImageType GetImageTypeFromKeyType(string? keyType)
         {
-            switch (keyType.ToLowerInvariant())
+            if (!string.IsNullOrEmpty(keyType))
             {
-                case "poster":
-                case "season": return ImageType.Primary;
-                case "series":
-                case "seasonwide": return ImageType.Banner;
-                case "fanart": return ImageType.Backdrop;
-                default: throw new ArgumentException($"Invalid or unknown keytype: {keyType}", nameof(keyType));
+                switch (keyType.ToLowerInvariant())
+                {
+                    case "poster": return ImageType.Primary;
+                    case "banner": return ImageType.Banner;
+                    case "background": return ImageType.Backdrop;
+                    case "clearlogo": return ImageType.Logo;
+                    default: throw new ArgumentException($"Invalid or unknown keytype: {keyType}", nameof(keyType));
+                }
             }
+
+            throw new ArgumentException($"Null keytype");
         }
 
         /// <summary>
         /// Normalize language to tvdb format.
         /// </summary>
         /// <param name="language">Language.</param>
         /// <returns>Normalized language.</returns>
-        public static string? NormalizeLanguage(string? language)
+        public static string? NormalizeLanguageToTvdb(string? language)
         {
             if (string.IsNullOrWhiteSpace(language))
             {
                 return null;
             }
 
-            // pt-br is just pt to tvdb
-            return language.Split('-')[0].ToLowerInvariant();
+            // Unique case for zh-TW
+            if (language.Equals("zh-TW", StringComparison.OrdinalIgnoreCase))
+            {
+                return "zhtw";
+            }
+
+            // to (ISO 639-2)
+            return CultureInfo.GetCultureInfo(language.Split('-')[0].ToLowerInvariant()).ThreeLetterISOLanguageName;
```

Comment:
I'm pretty sure the `CultureInfo` object isn't populated when running in our docker container

Also, use `.LeftPart` if possible (part of Jellyfin.Extensions)

Reply:
Alright. There is a way I can do a lookup for the languages that Jellyfin support?

Reply:
Hmm, is there a way for me to access Emby.Server.Implementations.Localization.iso6392.txt from the plugin?

Reply:
Sadly I don't think so, that project isn't meant to be referenced by plugins
I'm not sure how to correct this though

Reply:
Guess I can just dupe the txt file and build reference from there?